src/Features/VisualBasic/Portable/BraceCompletion/BracketBraceCompletionService.vb

```diff
@@ -7,71 +7,75 @@ Imports System.Threading
 Imports Microsoft.CodeAnalysis
 Imports Microsoft.CodeAnalysis.BraceCompletion
 Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.LanguageServices
+Imports Microsoft.CodeAnalysis.Text
 Imports Microsoft.CodeAnalysis.VisualBasic
+Imports Microsoft.CodeAnalysis.VisualBasic.LanguageServices
 Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
-<Export(LanguageNames.VisualBasic, GetType(IBraceCompletionService)), [Shared]>
-Friend Class BracketBraceCompletionService
-    Inherits AbstractBraceCompletionService
+Namespace Microsoft.CodeAnalysis.VisualBasic.BraceCompletion
```

Comment:
guess we just forgot namespaces here in the VB code.

Reply:
oops 🤦